client/my-sites/customer-home/main.jsx

```diff
@@ -86,16 +92,19 @@ class Home extends Component {
 			);
 		}
 
-		const { siteId, hasChecklistData, shouldRenderChecklist } = this.props;
+		const { siteId, hasChecklistData, isChecklistComplete } = this.props;
 
 		return (
 			<Main className="customer-home__main is-wide-layout">
 				<PageViewTracker path={ `/home/:site` } title={ translate( 'Customer Home' ) } />
 				<DocumentHead title={ translate( 'Customer Home' ) } />
 				<SidebarNavigation />
 				{ siteId && ! hasChecklistData && <QuerySiteChecklist siteId={ siteId } /> }
-				{ hasChecklistData &&
-					( shouldRenderChecklist ? this.renderChecklist() : this.renderCustomerHome() ) }
+				{ null !== isChecklistComplete && (
```

Comment:
Should we use `hasChecklistData` here? It feels more descriptive, but there might be something else in this logic.